packages/react-router/src/outlets.ts

```diff
@@ -0,0 +1,11 @@
+import type { Route } from "./routeRegistry.ts";
+
+export const ManagedRoutesOutletName = "__squide-managed-routes-outlet__";
+
+export const ManagedRoutes: Route = {
```

Comment:
ManagerRoute if there is only one

Reply:
If you are planning for the future and you think more routes are coming, you can change the constant for an array. It's either we plan for multiple or we plan for one and refactor when there is more

Reply:
ya the array part was dumb, i see that it's like an outlet. 

It feels  like there is an inconsistent naming here, 

it's a singular route, that is a placeholder, that we pluralized because this outlet/placeholder can be replaced by multiple routes, but then right after we have the method isManagedRoutesOutletRoute

which refers as the ManagedRoutes as ManagedRoutesOutlet. 

Should ManagerRoutes be ManagedRoutesOutlet?

Reply:
Ths is in fact what I had first but then renamed it to `ManagedRoutes` because I thought it could be confusing for the users if I reuse the "outlet" term since it's a concept/term that is already owned by React Router. This Squide concept is similar but also different.

What do you think?

Reply:
I am not 100% pleased with the `ManagedRoutes` name but I am also thinking that it's something that users will copy straight from the Getting Started documentaton (and event the CLI once available) twice per year without really giving it much tought so that might be good enough.